### DISCUSSION TITLE
Item Blacklist

### DISCUSSION THREAD

**Discussion by horsecraft** [Zrips/CMI#3678] [Feature Requests]
A way to blacklist the use of vanilla items. For example, if you do not want players ever using hoppers in-game you can add "hopper" to a blacklist. This blacklist will not allow the player to place the item on the ground. It will warn the player in chat that that item is not allowed on this server.

**Comment by CloudeLecaw** [comment]
this would be a great feature- but i think this list would be better to be able sepperating it by world
so you could define forbitten items by world.
We also have this problem with TNT and pistons where we dotn want users be able to place them in some worlds
